.github/workflows/ucai-integration-tests.yml

```diff
@@ -56,11 +61,16 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Install Core Library
+        run: |
+          pip install core/.
+      - name: Set PYTHONPATH for Core Library
+        run: |
+          echo "PYTHONPATH=$(pwd)/core/src:\$PYTHONPATH" >> $GITHUB_ENV
```

Comment:
This is to expose availability of the `test_utils` directory, which is not included in in the installation of the core package.